File: Python For Everyone, Horstmann/Chapter3/P3.15.py
# Write a program that reads in three floating­point numbers and prints the largest of
# the three inputs without using the  max function. For example:
#     Enter a number: 4
#     Enter a number: 9
#     Enter a number: 2.5
#     The largest number is 9.0

number1 = float(input("Enter a number: "))
number2 = float(input("Enter a number: "))
number3 = float(input("Enter a number: "))

largest = 0

if number1 >= number2 and number1 > number3:
    largest =  number1

elif number2 > number1 and number2 >= number3:
    largest = number2

else:
    largest = number3

print("Largest number is:", largest)

File: Big-Java-for-Everyone-Late-Objects-Horstmann/Chapter-3-Decisions/P3_15.java
/*Write a program that reads in three floating­point numbers and prints the largest of
 the three inputs without using the  max function. For example:
     Enter a number: 4
     Enter a number: 9
     Enter a number: 2.5
     The largest number is 9.0*/

import java.util.Scanner;

public class P3_15 {
	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		System.out.print("Enter three numbers seperating them by a space: ");
		double n1 = input.nextDouble();
		double n2 = input.nextDouble();
		double n3 = input.nextDouble();
		input.close();
		double largest;
		
		if (n1 >= n2 && n1 > n3) {
			largest = n1;
		} else if (n2 > n1 && n2 >= n3){
			largest = n2;
		} else {
			largest = n3;
		}
		System.out.println("Largest number: " + largest);
	}
}
